automatic/msys2/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,46 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = Split-Path $MyInvocation.MyCommand.Definition
+$is64     = (Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true'
+$dir_name = if ($is64) { 'msys64' } else { 'msys32' }
+
+$pp = Get-PackageParameters
+if (!$pp.InstallDir) { $pp.InstallDir =  "{0}\{1}" -f (Get-ToolsLocation), $dir_name  }
+$install_dir = $pp.InstallDir
```

Comment:
Since allowing custom installation directory, an uninstall script is needed to remove the path on uninstall.

Reply:
Done.

Reply:
The original package was not removing the directory. I added a message as was there before as I also think its safer to not delete it. Perhaps there is no harm in removing it but its a minor issue anyway so I opted for the safe side.